app/src/sharedTest/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsScreenTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.architecture.blueprints.todoapp.statistics
+
+import android.content.Context
+import androidx.activity.ComponentActivity
+import androidx.compose.material.Surface
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.MediumTest
+import com.example.android.architecture.blueprints.todoapp.R
+import com.example.android.architecture.blueprints.todoapp.data.Task
+import com.example.android.architecture.blueprints.todoapp.data.source.FakeRepository
+import com.example.android.architecture.blueprints.todoapp.util.saveTaskBlocking
+import com.google.accompanist.appcompattheme.AppCompatTheme
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Integration test for the statistics screen.
+ */
+@RunWith(AndroidJUnit4::class)
+@MediumTest
+@ExperimentalCoroutinesApi
+class StatisticsScreenTest {
+
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ComponentActivity>()
+
+    @Test
+    fun tasks_showsNonEmptyMessage() {
+        // Given some tasks
+        val repository = FakeRepository().apply {
```

Comment:
I'm wondering what our recommendation should be here. You can make a case for a unit test that calls the stateless StatisticsContent. It has less coverage but it's much more focused.